README.md

```diff
@@ -1 +1,63 @@
-# FunnySectors
\ No newline at end of file
+
+# Funny Sectors
+
+A lightweight plugin for managing large minecraft networks
+
+
+## Important note
+
+We do not have a working demo yet. [we](https://github.com/FunnyGuilds/FunnySectors/graphs/contributors) are actively working on creating one.
+
+
+## FAQ
+
+#### What are the priorities for the project?
+
+Well... We need a working demo. That is the most important thing for now. As [bopke](https://github.com/bopke) (project leader) [mentioned](https://github.com/FunnyGuilds/FunnySectors/issues/6) we need a working demo to avoid burnout.
```

Comment:
The real priority now is not even pursuing a demo, because we don't have anyone to demonstrate it to. The point of my statement is that we need a goal that we will see in reasonable distance. I don't think we need to mention priorities at all, because, after all, this is not a commercial project that we should really prioritize and I don't like viewing it as such. I believe that if we want to say that something is a priority, then it should be learning something now and having some fun doing it.

Reply:
I disagree. Let's start by saying that it does not matter if we do not have anyone to demonstrate our project to. Pushing a demo is important to get the motivation to contribute more time.
It is important to have a reasonable goal.

>  don't think we need to mention priorities at all, because, after all, this is not a commercial project 

You are wrong for couple of reasons. Fist of all we are targeting mostly wealthy audience. We can't be viewed as a joke. As such I do think that we should try to be professionals.
Second of all as You might know PR matters. That is why having clearly set priorities is important. Yes! This is an open source project but nonetheless it is a product that we are trying to promote.
If we don't care about the PR no one will be using our product.
Next let's talk about "this is not a commercial project". You are not wrong but if we create awesome, open source solution that will be easy to deploy and use we might get some attention from minecraft network owners.
They might be willing to help financially with the project. We want to be a real product and not a joke. It will be hard to change it later (priorities in the readme) later when a lot of contributes will create pull request etc.
Having this in mind will allow us potentially get some funding in the future.

> I believe that if we want to say that something is a priority, then it should be learning something now and having some fun doing it.

I agree! But we do not have to write this in the readme. Let's have fun and have professional start page at the same time! It is the best we can do for the project for reasons I explained above.